makefile

```diff
@@ -714,7 +718,7 @@ $(foreach a,$(ONEAPI.objs_a),$(eval $(call .compile.template.oneapi.ay,$a,$(ONEA
 $(foreach a,$(ONEAPI.objs_y),$(eval $(call .compile.template.oneapi.ay,$a,$(ONEAPI.tmpdir_y))))
 
 $(ONEAPI.tmpdir_y)/dll.res: $(VERSION_DATA_FILE)
-$(ONEAPI.tmpdir_y)/dll.res: RCOPT += $(addprefix -I, $(ONEAPI.incdirs))
+$(ONEAPI.tmpdir_y)/dll.res: RCOPT += $(addprefix -I, $(CORE.incdirs.common))
```

Comment:
Why not a `ONEAPI.incdirs.common`? I suppose ONEAPI.incdirs has a different meaning than CORE.incdirs...
Do you mean that now CORE belongs to both daal and oneDAL?

Reply:
`dll.res` requires headers from the old part only, but not from the new